cpptoml_vendor/package.xml

```diff
@@ -2,7 +2,7 @@
 <?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
 <package format="3">
   <name>cpptoml_vendor</name>
-  <version>0.1.1</version>
```

Comment:
please revert this, it's not our package

Reply:
We need to define the version we are using right? Since it is not ours, we do not define the version. Ah, I see. it is ours since we wrap the implementation.